docs/sys_modules_mock.py

```diff
@@ -0,0 +1,114 @@
+import sys
+from unittest.mock import MagicMock
+import json
+from pathlib import Path
+
+mockers = json.load(open(Path(__file__).parent.resolve() / "mocked_libs.json"))
+
+
+class DocsMocker(MagicMock):
```

Comment:
Would love to have some docstrings in this file to make it a bit easier to understand